### PR TITLE
feat: Add a bug/issue tracking list view

### PR DESCRIPTION
**Pull request by pauldambra**
See #team-arubug

Adds the issue tracking list view for logged in posthog users
Allows configuration of which event is reported on for issue tracking

### CODE REVIEW

File: frontend/src/lib/constants.tsx [hunk 1]
```diff
@@ -146,6 +146,7 @@ export const FEATURE_FLAGS = {
     RECORDINGS_LIST_V2: 'recordings-list-v2-enabled', // owner: #team-session-recordings
     HOGQL: 'hogql', // owner: #team-product-analytics
     NOTEBOOKS: 'notebooks', // owner: #team-session-recordings
+    ARUBUG: 'arubug', // owner: @team-arubug
```
Comment (tiina303) [inline frontend/src/lib/constants.tsx:149]:
```suggestion
    ARUBUG: 'arubug', // owner: #team-arubug
```

**Comment by pauldambra** [comment]
@mariusandra @thmsobrmlr I'm changing Query to support some things for hackathon

Want to review?

File: frontend/src/scenes/issues/Issues.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                                order: 0,
+                            },
+                        ],
+                        display: ChartDisplayType.ActionsLineGraph,
+                        interval: 'day',
+                        properties: query.properties,
+                    },
+                }}
+                context={{
+                    emptyStateHeading: 'Yay! No issues to report!',
```
Comment (tiina303) [inline frontend/src/scenes/issues/Issues.tsx:90]:
```suggestion
                    emptyStateHeading: 'Yay! No issues found!',
```

File: frontend/src/scenes/issues/Issues.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                    emptyStateDetail: '',
+                }}
+            />
+            <LemonDivider />
+            <h2>Open issues</h2>
+            <Query
+                query={tableQuery}
+                context={{ emptyStateHeading: 'Yay! No issues to report', emptyStateDetail: '' }}
```
Comment (tiina303) [inline frontend/src/scenes/issues/Issues.tsx:98]:
```suggestion
                context={{ emptyStateHeading: 'Yay! No issues found', emptyStateDetail: '' }}
```

**Review by tiina303 — APPROVED** [review]
🙌

**Pull request merged**